qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -695,3 +696,27 @@ def _set_sensitivity(self, s: float) -> int:
             return self._VOLT_TO_N[s]
         else:
             return self._CURR_TO_N[s]
+
+    def autorange(self, max_changes: int = 1) -> None:
+        """
+        Automatically changes the sensitivity of the instrument according to
+        the R value and defined max_changes.
```

Comment:
You need a blank line before the `Args:` line to get the docs to build correctly

Reply:
OOOPS
I missed that. Fix is comming :)
Thanks